community.md

```diff
@@ -1,9 +1,20 @@
 # Community
 
+## Slack
+
 The OpenFisca community gathers around a [Slack space](https://openfisca.slack.com), which you can ask to join by sending a mail to [contact@openfisca.org](mailto:contact@openfisca.org?subject=Slack).
 
 This space provides both community and official support, and centralises all countries’ channels.
 
+### Channels and naming conventions
+
+In order to increase discoverability of channels and ease navigation, the following prefixes are used:
+
```

Comment:
It seems to me it might be more interesting to have all channel concerning one country have a shared root : 
`france`
`france-goup-msa`
`france-goup-gouv`
`france-ext-paris`
It also prevent from having a country name on the one hand (france) and a contraction of the name (fr)
...

Reply:
Three concerns that made me not do that choice:

1. This works for more “well-known” countries names, while I am not certain it is obvious that `aotearoa`, for example, is a country package name.
2. This relies on country names being short. `nouvelle_caledonie` takes up 18 characters. It would be impossible to use any suffix on such a name in the 21 characters limit for Slack channels names.
3. I don't believe the incentive is for most contributors is to join all the channels of one country, or to join several countries. I would expect most users to join `general`, `random`, `support`, `share-news`, `country-CCC`, `group-CC-GGG` and possibly `ext-CC-EEE`. Namespacing by country looks more like an optimisation for people who subscribe to several countries than for most users.

Is this blocking for you?

Reply:
IMO, most contributors will work on one country package (with maybe extensions and group from that country) and be interested in shared ressources.
As slack channels are alphabetically ordered, they would see everything that is going on that affect their country package work in one block.
I think it would be more pleasing to have : 
```
france-country
france-ext-Paris
france-ext-brest
france-group-gouv
of-core
of-legislation-explorer
share-knowlege
share news
```
Rather than
```
country-france
ext-fr-Paris
ext-fr-brest
group-fr-gouv
of-core
of-legislation-explorer
share-knowlege
share news
```

On the `well know` factor of the country package name, I feel both namespace allow for deductive discoverability: Either its `country-aotearoa` or `aotearoa-country`

On length : I think longer country names (e.g.Liechtenstein) should pick one alternative writing (lie) to share across all dedicated channels. I feel that having both long name (e.g. france) and short name (e.g. fr) can become confusing if you are working on several country packages.

None of this is blocking :)